west.yml

```diff
@@ -152,7 +152,7 @@ manifest:
       groups:
         - hal
     - name: hal_espressif
-      revision: b3cb13f06586543ae1d156dc3e84b2fd3dc2f1fb
+      revision: be4d0fed7d40dd38ea81075734202eb77a122b75
```

Comment:
this patch should be first (but also feel free to send this separately if you prefer)

Reply:
this patch as the first commit really makes sense